app/script/main/app.js

```diff
@@ -125,6 +125,10 @@ z.main.App = class App {
       repositories.user
     );
 
+    repositories.integration = new z.integration.IntegrationRepository(
+      this.service.integration,
+      repositories.conversation
+    );
```

Comment:
No line removed or changed just added? You must be adding it twice then. I put them in alphabetical order.